bindata/linuxptp/ptp-daemon.yaml

```diff
@@ -173,6 +173,28 @@ spec:
       targetPort: https
   sessionAffinity: None
   type: ClusterIP
+  {{ if (eq .EnableEventPublisher true) }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    prometheus.io/scrape: "false"
+    service.beta.openshift.io/serving-cert-secret-name: linuxptp-daemon-secret
+  labels:
+    app: linuxptp-daemon
+  name:  ptp-event-publisher-service
+  namespace: openshift-ptp
+spec:
+  selector:
+    app: linuxptp-daemon
+  clusterIP: None
+  ports:
+    - name: publisher-port
+      port: 9086
```

Comment:
Why port 9086?

Reply:
changed it to 9043, its the port that is running default publisher service (HTP  event Publisher service)
I just picked one that would conflict with the one most used